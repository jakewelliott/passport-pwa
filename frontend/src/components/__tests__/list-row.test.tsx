import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import ListRow from '../list-row';

describe('ListRow', () => {
    it('matches snapshot', () => {
        const { container } = render(
            <ListRow>
                <div>Test Content</div>
            </ListRow>,
        );
        expect(container).toMatchSnapshot();
    });
});
