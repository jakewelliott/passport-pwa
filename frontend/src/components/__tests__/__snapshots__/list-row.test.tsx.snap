// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ListRow > matches snapshot 1`] = `
<div>
  <div
    class="flex items-center bg-supporting-lightblue p-3 shadow-md"
    style="border-radius: 15px;"
  >
    <div>
      Test Content
    </div>
  </div>
</div>
`;
